friends/src/App.css

```diff
@@ -0,0 +1,32 @@
+.App {
```

Comment:
You can remove this CSS since it came with create-react-app and you are not using it for your application